keps/1282-pods-ready-requeue-strategy/README.md

```diff
@@ -56,6 +56,22 @@ Consider the following scenario:
 
 In this case, the administrator would like the evicted workload to be requeued as soon as possible on the newly available capacity.
 
+#### Story 2
+
+In the story 1 scenario, when we set `waitForPodsReady.requeuingTimestamp=Eviction`, 
```

Comment:
I am guessing this should be `waitForPodsReady.requeuingTimestamp=Creation` ... which would preserve the pre-eviction ordering (i.e. put the evicted workload at the front of the queue again).

Reply:
Oops, it's a great catch! Thank you!

Reply:
Done.